src/order-history/OrderHistoryPage.test.jsx

```diff
@@ -24,19 +20,18 @@ global.matchMedia = (media) => ({
   matches: true,
 });
 
+const matchSnapshot = (store) => {
+  const { container } = render(
+    <ConnectedOrderHistoryPage {...requiredOrderHistoryPageProps} />,
+    store,
```

Comment:
I think we have pre-configured the redux store can we avoid passing it with every component render?